### PR TITLE
First commit

### PR DESCRIPTION
**Pull request by amatthews2020**
Updated to be a react app with typscript.  Created home page and explore page.  Idea is that the explore page displays users posts.  Home page displays explore page with additional button to navigate to creating a post.  Also implemented the Log in page but have not added the functionality to actually log in yet.

### CODE REVIEW

File: wd-final-project/src/WebApp/ExplorePage/index.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            <h1 id="wd-explore-title">Explore</h1> <hr />
+            <h2 id="wd-explore-recents-posts">Recent Posts</h2> <hr />
+            <div id="wd-explore-posts" className="row"></div>
+                <div id="wd-dashboard-courses" className="row">
+                    <div className="row row-cols-1 row-cols-md-5 g-4">
+                    <div className="wd-dashboard-course col" style={{ width: "300px" }}>
+                        <div className="card rounded-3 overflow-hidden">
+                            <Link to={{/* Fill later*/}}
+                                className="wd-dashboard-course-link text-decoration-none text-dark" >
+                            <img src={post.image} width="100%" height={160} />
```
Comment (ajohnson710) [inline wd-final-project/src/WebApp/ExplorePage/index.tsx]:
```suggestion
                            <img src={post.image} width="100%" height={160} alt="something"/>
```
Add alt text to fix linting issue

File: wd-final-project/src/WebApp/Home/PostControls.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import { FaPlus } from "react-icons/fa6";
+import { useNavigate, useParams } from "react-router";
```
Comment (ajohnson710) [inline wd-final-project/src/WebApp/Home/PostControls.tsx]:
```suggestion
import { useNavigate } from "react-router";
```
Delete UseParams

File: wd-final-project/src/WebApp/Navigation.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import { Link, useLocation } from "react-router-dom";
+
+export default function WebAppNavigation() {
+  const { pathname } = useLocation();
```
Comment (ajohnson710) [inline wd-final-project/src/WebApp/Navigation.tsx]:
```suggestion

```
delete line

File: wd-final-project/src/WebApp/Navigation.tsx [hunk 4]
```diff
@@ -0,0 +1,38 @@
+import { Link, useLocation } from "react-router-dom";
```
Comment (ajohnson710) [inline wd-final-project/src/WebApp/Navigation.tsx]:
```suggestion
import { Link } from "react-router-dom";
```
remove import

**Review by ajohnson710 — COMMENTED** [review]
Thanks for doing this, just a couple linting issues that shouldn't be lingering in a production build, but thats all. If you download eslint on vscode, it'll catch them for you.

**Review by amatthews2020 — COMMENTED** [review]
I think everything looks good

**Review by ajohnson710 — APPROVED** [review]

**Pull request merged**